docs/pages/database-access/rbac.mdx

```diff
@@ -138,3 +138,213 @@ spec:
     db_users: ["{{internal.db_users}}"]
     db_names: ["{{internal.db_names}}"]
 ```
+
+## Database object import rules
```

Comment:
The old intro part is missing entirely, and the name is gone too.

> **Database Access Controls** is a Teleport feature that lets you configure
role-based access controls for database objects. A **database
object** can be a table, view, or stored procedure. With Database Access Controls, you can ensure that users only have permissions to manage the data
they need.
>
> You can define **import rules** that instruct the Teleport Database Service to
apply labels to database objects imported from databases that match labels configured within the import rule. When a user connects to a database, the Database Service selectively grants permissions by checking database object labels against the user's Teleport roles.
>
> The Database Service grants object-level permissions for the duration of a
connection and revokes them automatically when the connection ends.

I think we should still use a separate name for this feature along with a brief intro section.

Reply:
I found it difficult to use the term "Database Access Controls" to describe restrictions on access to database objects, since in the context of this guide, it also seems to refer to the use of roles to restrict access to databases in general. In other words, the "Database Role-Based Access Controls" guide would both describe Database Access Controls, which have to do with database objects, and other role fields that do not.

Would it make sense to include this intro for the entire guide and encompass non-object-related role fields within "Database Access Controls"?

Reply:
I think it is worthwhile to highlight this feature in some way; the "Database Access Controls" name was picked by the product team to that end. @r0mant your thoughts on this?

Reply:
I think that Database Access Controls is the overall name for the functionality that includes both "old style" database server based and object based levels of granularity. Agree about merging the guides into one, from a user's standpoint, I don't think we want them to think about those as two different features.

Having a some kind of intro that explains that with Database Access Controls you can control access to both entire database servers as well as individual objects within those database servers sounds like a good idea though.